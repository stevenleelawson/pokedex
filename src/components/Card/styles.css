.card {
  height: 3em;
  width: 8em;
  background-color: grey;
  margin: 2em;
}

.card h1 {
  pointer-events: none;
}
