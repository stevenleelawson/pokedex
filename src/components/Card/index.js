import React from 'react';
import './styles.css';

const Card = ({name, id, pokeArray}) => {
  console.log('card',pokeArray)
  return (
    <div className='card'>
      <h1>{name}</h1>
    </div>
  )
}

export default Card
