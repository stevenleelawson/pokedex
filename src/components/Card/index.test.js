import React from 'react';
import { shallow } from 'enzyme';
import Card from './index';

describe('Card', () => {
  let wrapper;
  it('should match the snapshot', () => {
    wrapper = shallow(<Card />);
    expect(wrapper).toMatchSnapshot();
  });
})
