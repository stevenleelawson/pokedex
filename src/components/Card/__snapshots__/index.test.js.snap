// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Card />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1 />,
      "className": "card",
      "id": undefined,
      "onClick": undefined,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": undefined,
      },
      "ref": null,
      "rendered": null,
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1 />,
        "className": "card",
        "id": undefined,
        "onClick": undefined,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
